pythreepio/pythreepio/utils.py

```diff
@@ -1,11 +1,23 @@
+# importlib.resources is imported  to provide access to resources within packages
+
 try:
     import importlib.resources as pkg_resources
+    # for easier accessibility  importlib_resources can be refered to as pkg_resources
+
 except ImportError:
     import importlib_resources as pkg_resources
+    # except statement here represents importlib.resources in python3.6 and lower.   
+
 import json
 import static
+# static is imported to Serve static or templated content via WSGI 
+
 
 
 def get_mapped_commands() -> dict:
-    json_txt = pkg_resources.read_text(static, "mapped_commands_full.json")
-    return json.loads(json_txt)
+  """
+     this function is used to read data from "mapped_commands_full.json" file and returns an  dictionary  based on JSON file
```

Comment:
@somiljain7 It looks like this still isn't following the styleguide for docstrings. You can see the below for something that would fit the styleguide.

```python
def fetch_bigtable_rows(big_table, keys, other_silly_variable=None):
    """Fetches rows from a Bigtable.

    Retrieves rows pertaining to the given keys from the Table instance
    represented by big_table.  Silly things may happen if
    other_silly_variable is not None.

    Args:
        big_table: An open Bigtable Table instance.
        keys: A sequence of strings representing the key of each table row
            to fetch.
        other_silly_variable: Another optional variable, that has a much
            longer name than the other args, and which does nothing.

    Returns:
        A dict mapping keys to the corresponding table row data
        fetched. Each row is represented as a tuple of strings. For
        example:

        {'Serak': ('Rigel VII', 'Preparer'),
         'Zim': ('Irk', 'Invader'),
         'Lrrr': ('Omicron Persei 8', 'Emperor')}

        If a key from the keys argument is missing from the dictionary,
        then that row was not found in the table.

    Raises:
        IOError: An error occurred accessing the bigtable.Table object.
    """
```